prov/gni/src/gnix_util.c

```diff
@@ -161,3 +161,29 @@ int gnixu_get_rdma_credentials(void *addr, uint8_t *ptag, uint32_t *cookie)
 	}
 	return ret;
 }
+
+#define NUM_GNI_RC (GNI_RC_ERROR_NOMEM+1)
+static int gnix_rc_table[NUM_GNI_RC] = {
+	FI_SUCCESS, /* GNI_RC_SUCCESS = 0 */
```

Comment:
Do you have access to the GNI error codes here? I think it might improve clarity if you change this to be something like: 
```
static int gnix_rc_table[NUM_GNI_RC] = {
    [GNI_RC_SUCCESS] = FI_SUCCESS,
    [GNI_RC_NOT_DONE] = -FI_EOPBADSTATE,
    ...
};
```